src/lib/Libifl/pbsD_connect.c

```diff
@@ -525,18 +558,16 @@ __pbs_connect(char *server)
 }
 
 /**
- * @brief
- *	-send close connection batch request
+ * @brief	Helper function for __pbs_disconnect
  *
- * @param[in] connect - socket descriptor
+ * @param[in]	connect - connection to disconnect
  *
  * @return	int
  * @retval	0	success
  * @retval	-1	error
- *
  */
-int
-__pbs_disconnect(int connect)
+static int
+__pbs_disconnect_2(int connect)
```

Comment:
perhaps a more meaningful name is called for - how about disconnect_from_server() (in synergy with the corresponding connect function which is called connect_to_server())